di_website/datasection/migrations/0023_auto_20191121_0906.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.4 on 2019-11-21 09:06
```

Comment:
Could you remove these all migrations @davidebukali ? I'm afraid they'll conflict with the current ones.

Reply:
@akmiller01 deleted all conflicting migrations